src/hotspot/os/posix/os_posix.cpp

```diff
@@ -445,7 +445,7 @@ void os::Posix::print_load_average(outputStream* st) {
 // for reboot at least on my test machines
 void os::Posix::print_uptime_info(outputStream* st) {
   int bootsec = -1;
-  int currsec = time(nullptr);
+  int64_t currsec = time(nullptr);
```

Comment:
We should probably cast here even if the compiler doesn't complain, as we don't know what a time_t actually is.

Reply:
seeing if time_t will compile, that would be better.